libs/utils/utils.js

```diff
@@ -179,6 +183,9 @@ export const [setConfig, updateConfig, getConfig] = (() => {
       config.base = config.miloLibs || config.codeRoot;
       config.locale = pathname ? getLocale(conf.locales, pathname) : getLocale(conf.locales);
       config.autoBlocks = conf.autoBlocks ? [...AUTO_BLOCKS, ...conf.autoBlocks] : AUTO_BLOCKS;
+      config.doNotInline = conf.doNotInline
+        ? [...DO_NOT_INLINE, ...conf.doNotInline]
```

Comment:
Couldn't this lead to duplicates in the resulting array? Maybe something like `config.doNotInline = [...new Set([...DO_NOT_INLINE, ...conf.doNotInline || []])]` might do the trick?

Reply:
Consumers will never have blocks that have the same name as a Milo block since the Milo block will take precedence and the consumer block will be ignored.